src/class-wpml-elementor-woocommerce-hooks-factory.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+class WPML_Elementor_WooCommerce_Hooks_Factory implements IWPML_Backend_Action_Loader {
```

Comment:
Don't we need it also on the frontend @adrianosferreira?

Reply:
@strategio for now no, it is only needed when we are adding new widgets. It happens on Elementor editor which runs on WP admin scope.